tests/e2e/tracer_shutdown/tracer_shutdown_test.py

```diff
@@ -0,0 +1,32 @@
+import os
+import signal
+import subprocess
+import time
+import unittest
+import uuid
+
+from autoblocks._impl.api.client import AutoblocksAPIClient
+
+client = AutoblocksAPIClient()
+
+
+class TestMainScript(unittest.TestCase):
```

Comment:
can you just add this to the existing tests/e2e file? also write it in pytest syntax